src/containers/CodeContainer.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect } from 'react';
+import styled from 'styled-components';
+import { connect } from 'react-redux';
+import * as actions from '../actions/actionCreators';
+
+const Wrapper = styled.div`
+  border: 1px solid black;
+  width: 1150px;
+  height: 250px;
+  margin: 10px;
+  padding: 10px;
+  font-size: 30px;
+`;
+
+const mapStateToProps = store => ({
+  codeChallenge: store.games.codeChallenge,
+  charIndex: store.games.charIndex,
+});
+
+const mapDispatchToProps = dispatch => ({
+  moveChar: num => dispatch(actions.moveChar(num)),
+});
+
+const CodeContainer = ({ codeChallenge, charIndex, moveChar }) => {
+
+  const handleKeyPress = (e) => {
+    if (e.which === 32) e.preventDefault();
```

Comment:
why prevent default for a specific key?

Reply:
It is to prevent the "Tab" default function. The only way to treat a tab as two spaces is to prevent default for it.